src/translations/sqlb_it.ts

```diff
@@ -4457,32 +4458,32 @@ Se rispondi di no proveremo ad importare i dati del file SQL all&apos;interno de
     <message>
         <location filename="../MainWindow.cpp" line="327"/>
         <source>Window Layout</source>
-        <translation type="unfinished"></translation>
+        <translation>Disposizione finestra</translation>
     </message>
     <message>
         <location filename="../MainWindow.cpp" line="336"/>
         <source>Simplify Window Layout</source>
-        <translation type="unfinished"></translation>
+        <translation>Semplifica disposizione finestra</translation>
     </message>
     <message>
         <location filename="../MainWindow.cpp" line="337"/>
         <source>Shift+Alt+0</source>
-        <translation type="unfinished"></translation>
+        <translation>Shift+Alt+0</translation>
```

Comment:
We are asking to contributors not to translate shortcuts on the [wiki](https://github.com/sqlitebrowser/sqlitebrowser/wiki/Translations#dont-translate-key-names-in-shortcuts).

There was a discussion about this in PR #0000 some time ago, but in that case,
the shortcut didn't work and we wasn't sure if that was a Qt bug, so we proceeded the merge without deletion.

If possible, delete the shortcut translation, and if the shortcut doesn't work after deleting,
please tell us so we can just merge it without deletion.